src/views/graphs/FromMeToGmail.vue

```diff
@@ -1,31 +0,0 @@
-<template>
-  <div class="graph-enlarged">
-    <Header :title="headerText" :msg="descriptionText" />
-    <LineChart :title="label" :times="times" :fromMeToGmail="stats" />
-  </div>
-</template>
-
-<script>
-import Header from '../../components/Header.vue'
-import LineChart from '../../components/LineChart'
-import { useStore } from 'vuex'
-
-export default {
-  name: 'FromMeToGmail',
-  components: {
-    Header,
-    LineChart
-  },
-  setup() {
-    const store = useStore()
-    return {
```

Comment:
I think what I would have done differently is to create a generic EmailGraph.vue component with headerText, descriptionText, times, stats, and label as props.

I did notice that you setup each of these as its own routes so you would have needed individual components for each, so this is fine. :)

Reply:
LOL, yeah, I desperately wanted to create a generic EmailGraph.vue component as you mentioned. Is there even a way to conditionally render/set up routes? If there is a clean way, do let me know.

Reply:
You can specify props for each route. You can try something like this
1. Create EmailGraph.vue which takes in `graphType` as a prop
2. Register routes for each graph type (the 4 graphs you had). Pass `graphType` as a prop for each.
3. Since your store contains your email analytics, you can create a getter that returns your email stats keyed by `graphType`. In the `EmailGraph.vue` component, you can use the getter and get the stats using the `graphType` prop you had passed via the route

Reply:
pls don't worry about actually implementing it since your focus will be largely BE

Reply:
I like this idea, it really shows the power of the Vuex. Glad to be focusing on BE though haha, perhaps I'll start on the Twitter stats part of the tech ramp up soon.